lib/src/hydrated_bloc.dart

```diff
@@ -1,7 +1,7 @@
 import 'dart:async';
-import 'dart:convert';
 
 import 'package:meta/meta.dart';
+import 'package:flutter/foundation.dart';
```

Comment:
what is this import needed for?

Reply:
`freezed` uses it to make nicely readable objects in Flutter's devtools
This import is suggested in freezed docs

Reply:
just some random import lol